aura/src/Aura/Languages.hs

```diff
@@ -39,31 +39,19 @@ along with Aura.  If not, see <http://www.gnu.org/licenses/>.
 
 -}
 
-module Aura.Languages where
+module Aura.Languages
+    ( module Aura.Languages
```

Comment:
The problem is that we need to export `Language(..)` from `Base`, but if we specify that export, then all of the functions in `Aura.Languages` are not exported. So we either need to list each function or have the module export itself. The way I have it, `Fields` is not exported at all.

We could always go the route of moving all the translations to another module and keeping `Languages.hs` slim to avoid the self-export.